x-pack/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_rules_client.ts

```diff
@@ -395,3 +407,89 @@ const conditions = {
     return [this.isFullyTranslated(), this.isNotInstalled()];
   },
 };
+
```

Comment:
This file is starting to grow too much.
Would it make sense to move these sorting helpers to a separate place? 
Maybe:
- `server/lib/siem_migrations/rules/data/rule_migrations_data_rules_client/sort.ts`
And the `conditions` object to:
- `server/lib/siem_migrations/rules/data/rule_migrations_data_rules_client/search.ts`

Reply:
done